### DISCUSSION TITLE
QNAP packages: native or not?

### DISCUSSION THREAD

**Discussion by OneCDOnly** [nzbgetcom/nzbget#249] [Q&A] [answered]
Hello, what is the difference between the native and non-native QPKGs in each release please? When should I use one instead of the other? 

Thank you. 🤓

**Comment by phnzb** [comment, marked as answer]
@OneCDOnly 
- native qpkg packages - packages built with QNAP toolchains (only x86/x86_64/arm_64 QNAP architectures)
- non-native - buildroot qpkg, repacked from linux installer (all QNAP architectures)
We plan in v25 switch to C++17 and native qpkg packages support can be dropped due to old GCC in QNAP native toolchains. I think using non-native packages is preferable.

**Reply by OneCDOnly** [reply]
Makes sense, thank you. 🤓
